event/feed.go

```diff
@@ -207,12 +219,17 @@ func (sub *feedSub) Err() <-chan error {
 	return sub.err
 }
 
-type caseList []reflect.SelectCase
+type caseOption struct {
+	trySendOnly bool // Should this case only allow TrySend(), or is blocking Send() allowed
```

Comment:
Remove comment

Reply:
I was trying to describe what the `trySendOnly` option does. Should I move the comment above the struct?